src/Microsoft.Diagnostics.EventFlow.Outputs.EventHub/Microsoft.Diagnostics.EventFlow.Outputs.EventHub.csproj

```diff
@@ -20,6 +20,7 @@
 
   <ItemGroup>
```

Comment:
This, and the other outputs that will be using the new project, will also need a version change. A patch level upgrade is all that is needed probably.